coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -131,14 +144,32 @@
                   </v-menu>
                 </template>
 
-                <v-btn v-else-if="isPending(item)"
-                  class="btn-resume-payment"
-                  color="primary"
-                  :disabled="!item.enabled"
-                  @click.native.stop="doResumePayment(item)"
-                >
-                  <span>Resume Payment</span>
-                </v-btn>
+                 <template v-else-if="isPending(item)">
+                  <v-btn class="btn-resume-payment"
+                    color="primary"
+                    :disabled="!item.enabled"
+                    @click.native.stop="doResumePayment(item)"
+                    data-test-id="btn-resume-payment">
```

Comment:
We don't need `data-test-id` because we can already select this button via its class.
(Also, data-test-id might not be unique if there are several pending items.)